.stickler.yml

```diff
@@ -1,3 +1,7 @@
 linters:
   eslint:
-    config: './.eslintrc.json'
\ No newline at end of file
+    config: './.eslintrc.json'
+files:
+  ignore:
+      - 'src/*'
```

Comment:
Please, don't ignore the `src` folder. Since that is where the main code is, ignoring this is the same as not using a linter at all. Use this config instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml